lighthouse-cli/test/fixtures/static-server.js

```diff
@@ -150,12 +162,15 @@ const serverForOffline = http.createServer(requestHandler);
 serverForOnline.on('error', e => console.error(e.code, e));
 serverForOffline.on('error', e => console.error(e.code, e));
 
-
 // If called via `node static-server.js` then start listening, otherwise, just expose the servers
 if (require.main === module) {
   // Start listening
-  serverForOnline.listen(10200, 'localhost');
-  serverForOffline.listen(10503, 'localhost');
+  const onlinePort = 10200;
+  const offlinePort = 10503;
+  serverForOnline.listen(onlinePort, 'localhost');
+  serverForOffline.listen(offlinePort, 'localhost');
+  console.log(`online:  listening on http://localhost:${onlinePort}`);
+  console.log(`offline: listening on http://localhost:${offlinePort}`);
```

Comment:
there's really nothing offline or online about these, it's just how we use them for one(?) of the smoke tests (we also use them for same- and different-origin in other tests), but I'm not sure it matters what we call them :)